data/templates/suse-autoinst.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+
+  <partitioning config:type="list">
+    <drive>
+      <use>all</use>
+    </drive>
+  </partitioning>
+
+  <general>
+    <mode>
+      <!-- automation setting -->
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+
+  <!-- users settings -->
+  <users config:type="list">
+    <user>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>root</fullname>
+      <gid>0</gid>
+      <uid>0</uid>
+      <user_password><%=rootEncryptedPassword%></user_password>
+      <username>root</username>
+    </user>
+<% users.forEach(function(user) { %>
+    <user>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname><%=user.name%></fullname>
+      <user_password><%=user.encryptedPassword%></user_password>
+      <username><%=user.name%></username>
+    </user>
```

Comment:
For CentOS/RHLE/ESXi, it supports config each user's `uid` (`gid` is a miss), so this need a follow up action.